### PR TITLE
Spelling fixes in system test section

### PR DESCRIPTION
**Pull request by csmith608**
Fixed F` spelling in system test section and add try it yourself section to go along with the todo in the code.

### CODE REVIEW

File: docs/system-testing.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
+Before starting this guide, users should have the LedBlinking deployment running on their hardware and connected to the F´ GDS running on a development machine. If hardware is not available, this guide can be followed by running the LedBlinking deployment locally on a development machine instead.
 
-> Note: If running the LedBlinker deployment locally instead of on the intended hardware, make sure to rebuild fprime with stubbed GPIO drivers so the LedBlinker deployment doesn't attempt to write to physical GPIO ports. Regenerate the native deployment with `fprime-util generate -DFPRIME_USE_STUBBED_DRIVERS=ON`. MacOS defaults to stubbed drivers and does not require explicitly setting this option.
+> Note: If running the LedBlinker deployment locally instead of on the intended hardware, make sure to rebuild F´ with stubbed GPIO drivers so the LedBlinker deployment doesn't attempt to write to physical GPIO ports. Regenerate the native deployment with `fprime-util generate -DFPRIME_USE_STUBBED_DRIVERS=ON`. MacOS defaults to stubbed drivers and does not require explicitly setting this option.
 
 ## Intro to Fprime System Testing
```
Comment (kevin-f-ortega) [inline docs/system-testing.md]: Could you change Fprime to F Prime here?

File: docs/system-testing.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ $ pytest ../Components/Led/test/int/led_integration_tests.py
 
 ## Fprime System Testing Assertions
```
Comment (kevin-f-ortega) [inline docs/system-testing.md]: Same here, could you change Fprime to F Prime?

**Review by kevin-f-ortega — COMMENTED** [review]
Looks good, thank you. Just a few minor comments.

**Comment by kevin-f-ortega** [comment]
LGTM, thank you. Could you solve the conflicts?

**Review by thomas-bc — APPROVED** [review]

**Comment by thomas-bc** [comment]
The tutorial has been merged in a single file for the new website
I fixed some of the typo/styling issues in https://github.com/fprime-community/fprime-workshop-led-blinker/pull/99

For more changes (e.g. the system test stuff), we should open a new PR as this has diverged too much

**Pull request closed without merging**